### ISSUE TITLE
Omitting tsconfig option tsconfig-object fails

### ISSUE THREAD

**Issue by nsgundy** [TypeStrong/grunt-ts#302]
grunt-ts at 5.1.0

When I omit the tsconfig option within the tsconfig object:
```js
ts: {
      default: {
          tsconfig: {
            //tsconfig: 'tsconfig.json',
            ignoreFiles: false,
            updateFiles: true,
            overwriteFilesGlob: false
          },
          //src: ['<%= yeoman.app %>/scripts/**/*.ts', 'typings/tsd.d.ts']
      }
}
```
The result when running `grunt ts`:
```js
Error reading "undefined": {}
```
If I don't omit the tsconfig option, then everything is fine. According to the documentation using tsconfig inside the tsconfig object is optional and should default to `tsconfig.json` when omitted.

**Comment by nycdotnet** [comment]
Hi - sorry for the delay.  I've been very busy with a new job and baby and a few other projects.  I will try to look into this as soon as possible.

**Comment by nycdotnet** [comment]
Hi - I should get this into the next point release - thanks for the report.  Sorry I wanted to get 5.3.0 out the door and I got the other issue in.

**Comment by nsgundy** [comment]
Thanks for the update and all the great work you are doing!

**Comment by nycdotnet** [comment]
You're quite welcome.  I love feedback!  This should be fixed in 5.3.1 which I just published to npm.  Please re-open if there is an issue.  You're in the patch notes.  Thanks!
